src/components/TripCard/TripCard.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/macro'
+
+export default function TripCard({ city, startDate, endDate }) {
+  return (
+    <Card>
+      <Heading>{city}</Heading>
+      <DateWrapper>
+        <Date>{startDate.replace('-', ' ').replace('-', '/')}</Date>
+        <Date>{endDate.replace('-', ' ').replace('-', '/')}</Date>
+      </DateWrapper>
+    </Card>
+  )
+}
+
+const Card = styled.div`
+  background: rgba(250, 250, 250, 0.9);
+  height: 176px;
+  padding: 20px;
+  display: grid;
+  gap: 10px;
+  position: relative;
+`
+const Heading = styled.h3`
```

Comment:
Wenn ich das richtig sehe, ist h3 die einzige Überschrift dieser Komponente. Semantisch korrekt wäre dann h1 denke ich

Reply:
Ich habe eine h3 genommen, da ich das bezogen auf die App gesehen habe. Ich hab da wohl etwas in die Zukunft gedacht! Da ich morgen sowieso den Router einbaue und einzelne Seiten entstehen, werde ich das Styling bzw. die Elemente dann nochmal anpassen. Danke für das Feedback 👍🏻